services/movie.py

```diff
@@ -28,13 +33,16 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Avoid using mutable default arguments like lists. Instead, set the default to `None` and initialize the list inside the function if needed. This prevents potential issues with shared state between function calls.